charts/karavi-metrics-powerflex/nginx.conf

```diff
@@ -2,13 +2,16 @@ worker_processes  1;
 events {
     worker_connections  1024;
 }
+
+pid /tmp/nginx.pid;
+
 http {
     include       mime.types;
     default_type  application/octet-stream;
     sendfile        on;
     keepalive_timeout  65;
     server {
-      listen       443 ssl;
+      listen       8443 ssl;
```

Comment:
Can you also update https://github.com/dell/karavi/blob/main/docs/karavi-observability/README.md in the `karavi` repository to reference port 8443 for the otel-collector.

Reply:
Thanks, updated.